cmake/modules/FindNVJPEG.cmake

```diff
@@ -31,10 +31,14 @@ find_package_handle_standard_args(NVJPEG
     REQUIRED_VARS NVJPEG_INCLUDE_DIR NVJPEG_LIBRARY
     VERSION_VAR NVJPEG_VERSION)
 
+message(${CUDA_TOOLKIT_ROOT_DIR})
 if(NVJPEG_FOUND)
   # set includes and link libs for nvJpeg
   set(CMAKE_REQUIRED_INCLUDES ${CUDA_INCLUDE_DIRS})
-  set(CMAKE_REQUIRED_LIBRARIES ${NVJPEG_LIBRARY} "dl" "-pthread" "rt")
+  set(CMAKE_REQUIRED_LIBRARIES ${NVJPEG_LIBRARY} "dl" "-pthread" "rt" "-L${CUDA_TOOLKIT_ROOT_DIR}/lib64" "-lcudart_static" "-lculibos")
```

Comment:
to be tested that this doesn't break the old builds

Reply:
Testing it

Reply:
Works with old nvjpeg builds and 9.0 (in our build docker at least)